calendars/numpy.yaml

```
@@ -10,7 +10,7 @@ events:
       follow the link: https://hackmd.io/76o-IxCjQX2mOXO_wwkcpg
     begin: 2022-01-19 18:00:00 +00:00
     end: 2022-01-19 19:00:00 +00:00
-    url: https://berkeley.zoom.us/j/762261535
+    url:  https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
```

To make the linter happy
```suggestion
    url: https://us06web.zoom.us/j/83278611437?pwd=ekhoLzlHRjdWc0NOY2FQM0NPemdkZz09
```